test/connection.test.js

```diff
@@ -764,29 +773,29 @@ describe('connections:', function() {
       db2.close();
     });
 
-    it('cache connections to the same db', function() {
+    it('cache connections to the same db', function(done) {
       const db = start();
       const db2 = db.useDb(start.databases[1], { useCache: true });
       const db3 = db.useDb(start.databases[1], { useCache: true });
 
       assert.strictEqual(db2, db3);
-      db.close();
+      db.close(done);
```

Comment:
Let's not add extra `done()` usage, we should avoid using `done()`. `return db.close()` should be fine because `db.close()` returns promise if no callback specified.